lib/errors.js

```diff
@@ -203,6 +203,27 @@ const codes = {
     TypeError
   ),
 
+  FST_ERR_LOG_INVALID_LOGGER_INSTANCE: createError(
+    'FST_ERR_LOG_INVALID_LOGGER_INSTANCE',
+    'loggerInstance only accepts a logger not a configuration object.',
```

Comment:
@Fdawgs 
Can you please check the wording. 

Also it seems wrong. We dont know if it is a configuration object. If I pass any other Object, e.g. an Error, I should also get this error, despite an Error is not a configuration object.